GenderPayGap.WebUI/Views/Admin/Home.cshtml

```diff
@@ -22,6 +22,7 @@
 <form method="GET" action="@(Url.Action("SearchGet", "AdminSearch"))">
     <div class="govuk-form-group">
         <label class="govuk-label" for="search-query">
+            <span class="govuk-visually-hidden">Search for</span>
             <b>Organisation:</b> current name, previous name, employer reference, company number
             <br/>
             <b>User:</b> name, email address

```

Comment:
This label actually came up when I visited DAC!

There's far too much text in this label.
The screen reader reads out all the test in this label every time the user focusses on this text box!

I think we should split this into:
* A label that says "Search for organisations and users"
* A hint that says the "You can search for organisations by: current name, previous name, employer reference, company number <br> or for users by: name, email address"
https://design-system.service.gov.uk/components/text-input/#hint-text